packages/axios-extension/src/abap/adt/discovery.ts

```diff
@@ -0,0 +1,106 @@
+import type { AbapServiceProviderExtension } from '../abap-service-provider-extension';
+import XmlParser from 'fast-xml-parser';
+import type { AdtCategory, AdtSchemaData } from '../types';
+import type { ServiceProvider } from 'index';
+import 'reflect-metadata';
+
+const ADT_DISCOVERY_SERVICE = '/sap/bc/adt/discovery';
```

Comment:
Does this service have a stable public API?

Reply:
Yes

Reply:
Can you also add a link to the public documentation then please?